pkg/acs/calacs/acsccd/doccd.c

```diff
@@ -42,25 +64,13 @@ int DoCCD (ACSInfo *acs_info) {
 
     extern int status;
 
-    SingleGroup * x;  /* used for both input and output */
-    ACSInfo * acs;    /* hold a copy of the acs_info struct for each extension */
-    int option = 0;
-    float meanblev;   /* mean value of overscan bias (for history) */
-    int driftcorr;    /* true means bias level was corrected for drift */
-    int done;         /* true means the input SingleGroup has been freed */
-    int sizex, sizey;  /* size of output image */
-
-    int i, j;  /* loop index */
-    int * overscan;
-    int * blevcorr;
     char buff[ACS_FITS_REC+1];
-    Bool subarray;
 
     int to_electrons(ACSInfo *, SingleGroup *);
     int doBias (ACSInfo *, SingleGroup *);
     int biasHistory (ACSInfo *, Hdr *);
     int doBlev (ACSInfo *, SingleGroup *, int, float *, int *, int *);
-    int bias_shift_corr(ACSInfo *, SingleGroup *, SingleGroup *);
+    int bias_shift_corr(ACSInfo *, int, ...);
```

Comment:
This should be in a header. It should also have a comment describing the variadic arg.